src/neuroglancer/ui/annotations.ts

```diff
@@ -461,7 +461,25 @@ export class AnnotationLayerView extends Tab {
 
   private render(index: number) {
     const {annotation, state} = this.listElements[index];
-    return this.makeAnnotationListElement(annotation, state);
+    const {layer, gridTemplate, globalDimensionIndices, localDimensionIndices} = this;
+    const [element, elementColumnWidths] = makeAnnotationListElement(
+        layer, annotation, state, gridTemplate, globalDimensionIndices, localDimensionIndices);
+    for (const [column, width] of elementColumnWidths.entries()) {
+      this.setColumnWidth(column, width);
+    }
+    element.addEventListener('mouseenter', () => {
+      this.displayState.hoverState.value = {
+        id: annotation.id,
+        partIndex: 0,
+        annotationLayerState: state,
+      };
+    });
+    const selectionState = this.selectedAnnotationState.value;
+    if (selectionState !== undefined && selectionState.annotationLayerState === state &&
+        selectionState.annotationId === annotation.id) {
+      element.classList.add('neuroglancer-annotation-selected');
+    }
+    return element;
```

Comment:
moved a few things outside of `makeAnnotationListElement` that were more closely related to the list